src/profile/voyages/tag/vsca.js

```diff
@@ -80,7 +87,124 @@ try{
         if (u.data.url.indexOf("http") !== 0 && u.data.url.indexOf("/") !== 0) {
           u.data.url = location.protocol + "//" + u.data.url;
         }
+        //START Customised VSCA tag call
+	window.vsca_pageTag = window.vsca_pageTag || {};
```

Comment:
It will be used in the script loaded by this tag  : https://secure.analytics.voyages-sncf.com/prod/2.1/agency-funnel/vsca.js

u.data.url contain => https://secure.analytics.voyages-sncf.com/prod/2.1/agency-funnel/vsca.js